bc_obps/registration/api/operation.py

```diff
@@ -29,27 +30,40 @@ def get_operation(request, operation_id: int):
 
 @router.post("/operations")
 def create_operation(request, payload: OperationIn):
-    fields_to_assign = ["operator", "naics_code", "naics_category"]
 
+    operation_related_fields: OperationIn = extract_fields_from_dict(
```

Comment:
bcghg_id is optional for anyone who didn't submit a report in 2022, so I don't think checking it will ensure that operations don't already exist. Where's that conversation with Hanna? I can bring this up there

Reply:
I haven't heard from Hanna/whoever she needs to ask yet, so for now I added the check if the bcghg_id isn't null